packages/select/src/SelectField.js

```diff
@@ -1,29 +1,31 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import { FormGroup, Feedback } from '@availity/form';
 import { Label } from 'reactstrap';
 import Select from './Select';
 
 const SelectField = ({
-  label,
-  labelHidden,
-  name,
   feedbackClass,
   groupClass,
+  label,
   labelClass,
+  labelHidden,
+  name,
   ...attributes
 }) => {
+  useEffect(() => {
+    if (attributes.id && attributes.id === name) {
+      console.warn(
+        "Warning: Using an 'id' equivalent to 'name' will result in an orphaned label and break 508 compliance."
+      );
+    }
+  }, [attributes.id, name]);
+
   let thisLabel = false;
   if (label) {
-    let forLabel = name;
-    if (attributes.inputId) {
-      forLabel = attributes.inputId;
-    } else {
-      attributes.inputId = name;
```

Comment:
Can we still provide the default setter for `inputId` when its value is undefined or will that break other flows?

Meaning
```js
attributes.inputId = attributes.inputId || name
```

Reply:
[React-select will set this by default](https://github.com/JedWatson/react-select/blob/master/packages/react-select/src/Select.js#L1404) so I think we are safe to leave this out of our code.

Reply:
`inputId` is good for use during unit tests to grab the input by an id that you know of.

It looks like the `inputId` in their case is just [incremented by a count](https://github.com/JedWatson/react-select/blob/master/packages/react-select/src/Select.js#L367).

So passing in a `name` prop to or component will guarantee the `inputId` is:
1 - Unique
2 - The inputId in `react-select` is always `${name}-input`

Reply:
Ah so actually it looks like we default `inputId` to `name` in [our Select component](https://github.com/Availity/availity-react/blob/master/packages/select/src/Select.js#L106-L108), so we should still be good.